Configuration/FlexForms/flexform_newsfilter12.xml

```diff
@@ -36,6 +36,17 @@
                 <minitems>0</minitems>
             </config>
         </settings.filterTags>
+
+        <!-- Session Filter -->
+        <settings.keepFilterOnReload>
+            <TCEforms>
```

Comment:
@cepheiVV, `TCEforms` isn't "compatible" (?) with TYPO3 12 that's why the other options in this file don't have it.